src/components/modules/banners/contributions/variantA/ContributionsBannerVariantA.tsx

```diff
@@ -0,0 +1,182 @@
+        <div css={variantAStyles.bodyAndHeading}>
+            <div css={variantAStyles.heading}>{cleanHeading}</div>
+            <div css={variantAStyles.body}>
+                <div css={variantAStyles.rule} />
+                <div css={[styles.copy, variantAStyles.copy]}>
+                    {cleanMessageText}
+                    {cleanHighlightedText && (
+                        <>
+                            {' '}
+                            <span css={styles.highlightedText}>{cleanHighlightedText}</span>
+                        </>
+                    )}
+                </div>
+            </div>
+        </div>
+    );
+
+    const buttons = (
+        <div css={variantAStyles.buttonsContainer}>
+            <ContributionsBannerCloseButton onCloseClick={onCloseClick} />
+            <div css={variantAStyles.ctaContainer}>
+                <ContributionsBannerCta
+                    onContributeClick={onContributeClick}
+                    ctaText={ctaText}
+                    ctaUrl={ctaUrl}
+                    stacked={true}
+                />
+            </div>
+        </div>
+    );
+
+    return (
+        <div css={styles.bannerContainer}>
+            <ContributionsBannerMobile
+                onCloseClick={onCloseClick}
+                onContributeClick={onContributeClick}
+                heading={cleanHeading}
+                messageText={cleanMobileMessageText || cleanMessageText}
+                highlightedText={cleanHighlightedText}
+                ctaUrl={ctaUrl}
+                ctaText={ctaText}
+            />
+
+            <Container cssOverrides={variantAStyles.columnsContainer}>
+                <div css={variantAStyles.tabletAndDesktop}>
+                    <Columns>
+                        <Column width={8 / columnCounts.tablet}>
+                            <BodyAndHeading />
+                        </Column>
+                        <Column width={4 / columnCounts.tablet}>{buttons}</Column>
+                    </Columns>
+                </div>
+                <div css={variantAStyles.leftCol}>
+                    <Columns>
+                        <Column width={2 / columnCounts.leftCol}> </Column>
+                        <Column width={8 / columnCounts.leftCol}>
+                            <BodyAndHeading />
+                        </Column>
+                        <Column width={4 / columnCounts.leftCol}>{buttons}</Column>
+                    </Columns>
+                </div>
+                <div css={variantAStyles.wide}>
+                    <Columns>
+                        <Column width={3 / columnCounts.wide}> </Column>
+                        <Column width={8 / columnCounts.wide}>
+                            <BodyAndHeading />
+                        </Column>
+                        <Column width={4 / columnCounts.wide}>{buttons}</Column>
+                        <Column width={1 / columnCounts.wide}> </Column>
+                    </Columns>
```

Comment:
Loving the use of columns though 👌 I feel like the `width` api is a little awkward to deal with. I wonder if it's worth chatting to Simon about? It feels like it's more natural to think in terms of how many grid columns you want to span

Reply:
Yes there's been some discussion about the api in the RR design channel, Simon is looking for feedback